src/apps/bridge/sensor_drivers/rbrCodaSensor.cpp

```diff
@@ -117,16 +117,19 @@ void RbrCodaSensor::aggregate(void) {
                                     .reading_count = 0,
                                     .sensor_type = BmRbrDataMsg::SensorType::UNKNOWN};
     aggs.sensor_type = rbrCodaGetSensorType(node_id);
+    latest_sensor_type = aggs.sensor_type;
```

Comment:
This member variable `latest_sensor_type` was being checked below but was never set.